fixtures/get-platform-proxy-remote-bindings/tests/local-bindings-only.test.ts

```diff
@@ -0,0 +1,30 @@
+import { describe, expect, test } from "vitest";
+import { getPlatformProxy } from "wrangler";
+import type { Ai } from "@cloudflare/workers-types/experimental";
+
+describe(
+	"getPlatformProxy - remote bindings with localBindingsOnly",
+	{ timeout: 50_000 },
+	() => {
+		test("getPlatformProxy works with remote bindings", async () => {
+			const { env, dispose } = await getPlatformProxy<{
+				AI: Ai;
+			}>({
+				configPath: "./wrangler.local-bindings-only.jsonc",
+				localBindingsOnly: true,
+			});
+
+			await expect(
+				(async () => {
+					await env.AI.run("@cf/meta/llama-3.1-8b-instruct-fp8", {
+						messages: [],
+					});
+				})()
```

Comment:
Does this need to be wrapped in a function?

Reply:
oh right, good callout 👍